subsys/logging/log_cmds.c

```diff
@@ -455,6 +479,7 @@ SHELL_STATIC_SUBCMD_SET_CREATE(sub_log_stat,
 	SHELL_COND_CMD_ARG(CONFIG_LOG_STRDUP_POOL_PROFILING, strdup_utilization,
 			NULL, "Get utilization of string duplicates pool",
 			cmd_log_strdup_utilization, 1, 0),
+	SHELL_CMD(mem, NULL, "Logger memory usage", cmd_log_memory_slabs),
```

Comment:
This command is valid only for deferred mode with logging v2. Please use
```
SHELL_COND_CMD(CONFIG_LOG_MODE_DEFERRED, ...)
```
Util utilization instrumentation is available for circular buffer used for v2.

Reply:
Fixed, and reformatted section of code based on new 100 column width.